jmbitcoin/jmbitcoin/secp256k1_main.py

```diff
@@ -103,7 +101,7 @@ def getG(compressed=True):
     representation of secp256k1 G
     """
     priv = "\x00"*31 + "\x01"
-    G = secp256k1.PrivateKey(priv, ctx=ctx).pubkey.serialize(compressed)
+    G = secp256k1.PrivateKey(priv, context=ctx).public_key.format(compressed)
```

Comment:
This comment applies to all uses of `context=ctx`:

Afaict (still reading), coincurve defaults to using its own global context object (in its `context.py`) which is flags_all as ours was. I think we can ditch our one to defer that control to the underlying binding (thus remove our ctx object and remove all these argument usages).

Reply:
Removed context arguments.